environment.yml

```diff
@@ -5,48 +5,46 @@ channels:
   - esri  # for pycocotools. Remove when https://github.com/conda-forge/pycocotools-feedstock/issues/16 is solved
   - open3d-admin  # for open3d
 dependencies:
-  - einops
-  - fiona>=1.5
-  - h5py
-  - numpy
+  - einops>=0.3
+  - fiona>=1.8
+  - h5py>=2.6
+  - numpy>=1.17.2
   - open3d>=0.11.2
   - pip
-  - protobuf<4
```

Comment:
No need to pin this here, it isn't used by CI so it won't cause us issues. If someone encounters an issue they can manually install an older protobuf.